src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -744,8 +753,17 @@ private void Dispose()
                 string bkptno = results.Results.FindString("bkptno");
                 ulong addr = cxt.pc ?? 0;
 
-                bool fContinue;
                 TupleValue frame = results.Results.TryFind<TupleValue>("frame");
+                if (frame != null && addr != 0)
+                {
+                    string sourceFile = frame.TryFindString("fullname");
+                    if (!String.IsNullOrEmpty(sourceFile))
+                    {
+                        await this.VerifySourceFileTimestamp(addr, sourceFile);
```

Comment:
This is also only going to do the verification in the leaf stack frame. We might want to check other frames on the call stack.

---
In reply to: [62338268](https://github.com/Microsoft/MIEngine/pull/308#discussion_r62338268) [](ancestors = 62338268)